zzzz_modular_occulus/code/modules/telescience/telepad_computer.dm

```diff
@@ -110,29 +113,53 @@
 		for(var/i in 1 to numPings)
 			var/turf/randTurf = get_random_turf_in_range(telegraph, 7, 1)
 			new /obj/effect/telesci_ping(randTurf)
-		playsound(telegraph,'zzzz_modular_occulus/sound/effects/telesci_ping.ogg',50,FALSE,7,is_global = FALSE, ignore_walls = TRUE)
+		playsound(telegraph,'zzzz_modular_occulus/sound/effects/telesci_ping.ogg',50,FALSE, 7, extrarange = 10, is_global = FALSE, ignore_walls = TRUE)
+		playsound(src,'zzzz_modular_occulus/sound/effects/telesci_process.ogg',50,FALSE, 7, is_global = FALSE)
 
 /obj/machinery/computer/telesci_console/proc/addLog(string)
 	teleLog = "\[[stationtime2text()]\] " + string + "<br>" + teleLog
 
 /obj/machinery/computer/telesci_console/proc/openPortal()
 	portalOpened = TRUE
 	telepad.openPortal(targetX, targetY, targetZ)
-	if(dangerous)
+	var/area/portalarea = get_area(get_turf(locate(targetX,targetY,targetZ)))
+	if(portalarea.tele_inhibited())
 		for(var/obj/machinery/telesci_relay/relay in telepad.relaysInUse)
 			relay.chanceExplode()
+		var/inhibitorExploded = FALSE
+		for(var/obj/machinery/telesci_inhibitor/blocker in portalarea.tele_inhibitors)
+			if(!istype(blocker))
+				continue
+			inhibitorExploded = TRUE
+			playsound(telegraph,'zzzz_modular_occulus/sound/effects/telesci_inhibitor_alarm.ogg', 80, FALSE, 7, extrarange = 10, is_global = FALSE, ignore_walls = TRUE)
+			blocker.visible_message(SPAN_DANGER("\The [src] sparks violently and begins to shake!"))
+			do_sparks(6, FALSE, get_turf(blocker))
+			addtimer(CALLBACK(blocker, /obj/machinery/telesci_inhibitor/proc/explode), 1 SECONDS)
```

Comment:
```suggestion
			addtimer(CALLBACK(blocker, /obj/machinery/telesci_inhibitor/proc/explode), 1 SECOND)
```
\>:C

Reply:
I did not know someone made a `SECOND` define specifically to allow for grammatically correct code, that's really funny.